sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/DropwizardLegacyHelper.java

```diff
@@ -0,0 +1,33 @@
+package org.sdase.commons.server.testing;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jersey.jackson.JacksonFeature;
+import javax.ws.rs.client.Client;
+import org.glassfish.jersey.client.ClientProperties;
+import org.glassfish.jersey.client.JerseyClientBuilder;
+
+public class DropwizardLegacyHelper {
```

Comment:
Imho that's a good way to mitigate the changes coming in with Dropwizard. Could you provide a helpful addition for the release notes that explain the bigger changes of this update?
As far as I can see, this would include 

- information about the circumstances where you need to switch the `DropwizardLegacyHelper`
- information about the changes related to the assertions in AssertJ

Even if we don't consider these changes as breaking, I would prefer to increase at least the minor version. I would never expect possible defects of my tests in a patch release. Is there a commit, you could change to feat?